faker/providers/person/th_TH/__init__.py

```diff
@@ -62,29 +68,87 @@ class Provider(PersonProvider):
-        "ชมศรี", "ตังคเสฐกุล", "โขวิฑูนกิต", "ทศพรพิทักษ์กุล", "วิมลโนต", "กิตติคุณ",
-        "เมธาวรกุล", "ปิตานุวัฒน์", "ภูศิลารุ่งเรือง", "ตุรงคินานนท์", "ชินะกิจประภา",
-        "โพธานันท์", "อยู่ประเสริฐ", "เมธาวรกุล", "เวทยศาสน์", "สุขสว่าง", "อเนกวรกุล",
-        "พิชญ์พันธ์เดชา", "สิทธิ์เสาวภาคย์", "สุระประจิต", "คงศรี", "ตรีเกษมมาด",
-        "หาบพนม",
-        "วรรณไพฑูลศรี", "วิญญูวานิชกุล", "พงศ์พาณิชย์", "เพิ่มชาติ", "ไชยหิรัญการณ์",
-        "ธารทนานนท์", "นรมน", "ประยูงหงส์", "เลิศสาธยานุศักดิ์", "ผลอวยพร", "ประคัลภวงศ์",
-        "ธิติภัทตรายันยง", "กฤตายานุกุล", "ศิริภัยบูล",)
+        "ดำริห์ชอบ", "ดวงทับทิม", "ดิศดใน", "ดัตพันธุ์", "ดุษฎีวนิช", "ดาบเพ็ชร์",
+        "ดิสกะประกาย", "เดชคุ้ม", "ดาวกระจาย", "ดาบเงิน", "ดีตพันธุ์", "ดุริยพันธุ์",
+        "ดาวอร่าม", "เดชวา", "ติระคมน์", "ไตรบรรพ", "ตระกูลบุญ", "เตมิยะเดช", "ตวงทอง",
+        "แต้กุล", "ตรีครุธพันธุ์", "ตัณฑนุช", "ตั้งกุลงาม", "ตระกูลไม้เรียง", "ตะละภัฏ",
+        "ตั้งเผ่า", "ตันเผ่า", "เตชะกำพุ", "ตัณสถิตย์", "ตราชู", "ตวันเยี่ยม",
+        "ตรีเภรินทร์",
+        "ตันตราจิณ", "ตั้งรบ", "ถนัดกลึง", "ถนอมมนุษย์", "ถนัดอักษร", "ถนัดการยนต์",
+        "ถนัดหัดถกรรม", "ถาวรายุศม์", "ถนัดการเขียน", "ถาวรรัตน", "ถนัดพิมพการ",
+        "ถ้วนศรี",
+        "ถนอมพล", "ถาวระวรณ์", "ถนัดรบ", "ถนอมกุลบุตร", "ถมปัด", "ถนัดภาษา",
+        "ถนัดเดินข่าว",
+        "ถนัดรักษา", "ถนิมมาศ", "ถมังรักษสัตว์", "แถมธน", "ถะเกิงชศ", "ถิรสวัสดิ์",
+        "ถนัดอาวุธ", "ถุงเงิน", "ทุมะบุตร์", "ทองประดิฐ", "ทรงโกมล", "ทัศนสุทธิ",
+        "ทหารแท้",
+        "ทองอยู่", "ทำประดู่", "ทรัพย์สาร", "ทีฆะ", "ทวนทอง", "ไทยสุชาต", "ทันยุค",
+        "ไทนิยม",
+        "ทองลาภ", "ทองเนื้อดี", "ทองแท้", "ทวีเดช", "ทับทิมไทย", "ทองสินธุ์", "ไทไชโย",
+        "ทวนไชย์", "ทองสุกเลิศ", "ไทยแท้", "ทองสีไพล", "ธรรมนิยม", "ธนประทีป", "ธรรมเมธา",
+        "ธุวะนุติ์", "ธาราธร", "ธีวร", "ธัญเสถียร", "ธนูปกรณ์", "เธียรายัน", "ธรรมฤดี",
+        "ธัญาโภชน์", "ธนรักษ์", "ธูปะวิโรจน์", "ธูปหอม", "แนวพนิช", "นักสำหรวจ",
+        "นฤทุกข์",
+        "นาฏคายี", "เนื่องนนท์", "น้ำทิพย์", "เนื้อนุ่ม", "นวลฉวี", "นาคพันธุ์", "นำธวัช",
+        "แน่นดุจป้อม", "เณรานุสนธิ์", "นามเสวตร", "นิษประปัญจ์", "นิระหานี", "นิลสุวรรณ์",
+        "นักรบ", "นิติสาขา", "นิยมธรรม", "นิลเสนา", "หนุนสุข", "นิลวิมล", "นฤภัย",
+        "นาถะเดชะ",
+        "นพตระกูล", "นิยมเซียม", "หนักแน่น", "นามขำ", "นานายน", "นับเนื่องนอ", "แนวพญา",
+        "นาถะพินธุ", "นิลสลัว", "นะวะมันดร", "นิลวรรณ", "เนตร์มณี", "นุตตาร",
+        "นิยมสำหรวจ",
+        "นกทอง", "นุชแนวนุ่ม", "นาคสุทิน", "นพคเชนทร์", "นาคะนคร", "นิละทัต", "นาควงษ์",
+        "นากกนก", "บุนยาภิสนท์", "บุญญาภิรมย์", "บุณยะภาชน์", "บินดี", "บุนยะศัพท์",
+        "บุนยะตีรณะ", "บุญญาไลย์", "บุญบำรุง",)
+
+    # thai suffix
+    # https://th.wikipedia.org/wiki/%E0%B8%99%E0%B8%B2%E0%B8%A1%E0%B8%AA%E0%B8%81%E0%B8%B8%E0%B8%A5%E0%B8%9E%E0%B8%A3%E0%B8%B0%E0%B8%A3%E0%B8%B2%E0%B8%8A%E0%B8%97%E0%B8%B2%E0%B8%99
```

Comment:
By translating the page with google, this appears to be a list of last names, not suffixes.